app/src/main/java/com/cornellappdev/uplift/ui/components/Home/HomeCard.kt

```diff
@@ -0,0 +1,128 @@
+            ) {
+                Column() {
+                    Column(modifier = Modifier.weight(3F)) {
+                        Image(
+                            modifier = Modifier.fillMaxWidth(),
+                            painter = painterResource(id=R.drawable.court_lines),
+                            contentDescription = "Gym Image",
+                            contentScale = ContentScale.FillWidth
+                        )
+
+                    }
+                    Column(modifier = Modifier.weight(2F)) {
+                        Row(
+                            modifier = Modifier.padding(5.dp)
+                        ) {
+                            Text(
+                                text =gymexample.name,
+                                fontSize = 20.sp,
+                                fontFamily = montserratFamily
+
+                                )
+                            Spacer(modifier = Modifier.weight(1F))
+                            Image(
+                                painter = painterResource(id = R.drawable.ic_dumbbell),
+                                contentDescription = "Dumbell",
+                            )
+                            Image(
+                                painter = painterResource(id = R.drawable.ic_bowling_pins),
+                                contentDescription = "Bowling Pins",
+                            )
+
+                        }
+                        Row(modifier = Modifier.padding(start = 5.dp, end = 5.dp)) {
+                            Text(
+                                text = "Open",
+                                fontSize = 12.sp,
+                                color = Color.Green,
+                                fontFamily = montserratFamily
+
+                                )
+                            Spacer(modifier = Modifier.width(10.dp))
+                            Text(
+                                text = "Closes at 6 pm",
+                                fontSize = 12.sp,
+                                fontFamily = montserratFamily
+                                )
+
+                        }
+                        Row(
+                            modifier = Modifier.padding(
+                                start = 5.dp,
+                                end = 5.dp
+                            )
+                        ) {
+                            Text(
+                                text = "Cramped",
+                                fontSize = 12.sp,
+                                color = Color.Red,
+                                fontFamily = montserratFamily
+                                )
+                            Spacer(modifier = Modifier.width(10.dp))
+                            Text(
+                                text = " 120/140",
+                                fontSize = 12.sp,
+                                fontFamily = montserratFamily
+                            )
```

Comment:
We will want to make the gym full-ness status dynamically update at some point, of course. I'll take care of this once we get to networking, so don't worry about it for this PR... but let's keep it in mind for the codebase in the future! (Maybe add a TODO here?)